src/features/content-explorer/messages.js

```diff
@@ -82,6 +82,28 @@ const messages = defineMessages({
         description: 'Text shown to indicate the number of items selected',
         id: 'boxui.contentExplorer.numSelected',
     },
+    numItemsSelected: {
+        defaultMessage: `
+            {numSelected, plural,
+                =0 {0 items selected}
+                one {1 item selected}
+                other {# items selected}
+            }
```

Comment:
woah neat!! i've never seen this before but if it works that's awesome

Reply:
Yeah! I saw this in the admin console repo messages files